src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -87,8 +87,8 @@ const WorkspaceContainer = ({
         div({ style: Style.noWrapEllipsis }, breadcrumbs),
         h2({ style: Style.breadcrumb.textUnderBreadcrumb }, [
           title || `${namespace}/${name}`,
-          workspace && !Utils.canWrite(workspace.accessLevel) && span({ style: { paddingLeft: '0.5rem', color: colors.dark(0.85) } }, '(read only)'),
-          workspace && workspace.workspace.isLocked && span({ style: { paddingLeft: '0.5rem', color: colors.dark(0.85) } }, '(locked)')
+          workspace && !Utils.canWrite(workspace.accessLevel) && span({ style: { paddingLeft: '0.5rem' } }, '(read only)'),
```

Comment:
This text is not legible and doesn't meet color contrast. Before:

![Workspaces ›](https://user-images.githubusercontent.com/484484/158704983-dc4f916d-abf9-4392-9122-7a2c3adb1e1f.png)

After:
![image](https://user-images.githubusercontent.com/484484/158705142-b91e77fd-e6e8-432c-b805-eda3c6796ef5.png)

Reply:
Oh wow - great catch! 

Does it need a UX ticket for follow-up in case they may know a better way of emphasizing the `read-only`ness?

Reply:
I personally don't think it's worth creating a UX ticket. I know Lou is looking at revamping this part of the UI. I'll send him a note about it.

Reply:
Lou came up with something much nicer-- https://broadworkbench.atlassian.net/browse/WOR-217